src/internal/ajax/AjaxResponse.ts

```diff
@@ -32,17 +33,15 @@ export class AjaxResponse<T> {
 
   /**
    * The total number of bytes loaded so far. To be used with {@link total} while
-   * calcating progress. (You will want to set {@link includeDownloadProgress} or {@link includeDownloadProgress})
-   *
-   * {@see {@link AjaxConfig}}
```

Comment:
This `@see` doesn't work here (it doesn't seem to be rendered anywhere on the page). For this reason, I moved it to line 16.